TuneUp/manifests/pkg.json

```diff
@@ -1,22 +1,22 @@
 {
-  "license": "",
-  "file_hash": null,
-  "name": "TuneUp",
-  "version": "1.0.1",
-  "description": "TuneUp is in beta and is a view extension for analyzing the performance of Dynamo graphs. TuneUp allows you to see overall graph execution time, per-node execution time, and other helpful information about what's happening under the hood, e.g. nodes run in the current execution v.s. nodes run in the previous execution (which are skipped this run for optimization/ caching)\r\nKnown issues:\r\n1. TuneUp does not work with .dyfs (custom nodes) yet.\r\n2. TuneUp binaries are not semantically versioned and are not intended to be built on top of as an API. Do not treat these binaries like DynamoCore.\r\n3. TuneUp requires Dynamo 2.5 or higher for access to new extension APIs.\r\n4. When user have TuneUp open, after switching workspace in Dynamo, the first graph run does not give execution time and nodes order.",
-  "group": "",
-  "keywords": [
-    "profiler",
-    "tuneup"
-  ],
-  "dependencies": [],
-  "host_dependencies": [],
-  "contents": "",
-  "engine_version": "2.5.0",
-  "engine": "dynamo",
-  "engine_metadata": "",
-  "site_url": "https://dynamobim.org/",
-  "repository_url": "https://github.com/DynamoDS/TuneUp",
-  "contains_binaries": true,
-  "node_libraries": []
+    "license": "",
+    "file_hash": null,
+    "name": "TuneUp",
+    "version": "1.0.6",
+    "description": "TuneUp is in beta and is a view extension for analyzing the performance of Dynamo graphs. TuneUp allows you to see overall graph execution time, per-node execution time, and other helpful information about what's happening under the hood, e.g. nodes run in the current execution v.s. nodes run in the previous execution (which are skipped this run for optimization/ caching). Read more here:https://dynamobim.org/tuneup-extension-explore-your-node-and-graph-execution-times/\r\nKnown issues:\r\n1. TuneUp does not work with .dyfs (custom nodes) yet.\r\n2. TuneUp binaries are not semantically versioned and are not intended to be built on top of as an API. Do not treat these binaries like DynamoCore.\r\n3. TuneUp requires Dynamo 2.5 or higher for access to new extension APIs.\r\n4. When user have TuneUp open, after switching workspace in Dynamo, the first graph run does not give execution time and nodes order.",
+    "group": "",
+    "keywords": [
+        "profiler",
+        "tuneup"
+    ],
+    "dependencies": [],
+    "host_dependencies": [],
+    "contents": "",
+    "engine_version": "2.5.0",
+    "engine": "dynamo",
+    "engine_metadata": "",
+    "site_url": "https://dynamobim.org/",
+    "repository_url": "https://github.com/DynamoDS/TuneUp",
+    "contains_binaries": true,
+    "node_libraries": []
```

Comment:
@mmisol Can you check your VSCode setting for what's used for tabbing? Spaces or tabs?

Reply:
Sorry, they were spaces but my VS decided to format with 4 of them instead of 2. Fixed it